js/jquery.flot.tooltip.js

```diff
@@ -90,48 +110,74 @@
             that.updateTooltipPosition(pos);
         }
 
+        function _handleHoverOnEmptyItem() {
+            const tooltipCount = Object.keys(that.tooltips).length;
+            if (tooltipCount === 2) {
+                if (that.tooltips[0].isSticky && that.tooltips[1].isSticky) {
```

Comment:
should add a comment here like if we have 2 tooltips and we have hovered over a non-target, we should hide the non-sticky tooltip